src/Umbraco.Web.UI.Client/src/views/content/content.edit.controller.js

```diff
@@ -24,7 +24,7 @@ function ContentEditController($scope, $routeParams, contentResource) {
     $scope.page = $routeParams.page;
     $scope.isNew = infiniteMode ? $scope.model.create : $routeParams.create;
     //load the default culture selected in the main tree if any
-    $scope.culture = $routeParams.cculture ? $routeParams.cculture : ($routeParams.mculture === "true");
+    $scope.culture = $routeParams.cculture ? $routeParams.cculture : $routeParams.mculture;
```

Comment:
Or simplified to the following :)

```
$scope.culture = $routeParams.cculture || $routeParams.mculture;
``

Reply:
@bjarnef true, and I prefer that syntax myself. But `$scope.culture = $routeParams.cculture ? $routeParams.cculture : $routeParams.mculture;` is used throughout the codebase as far as I can tell. Best be consistent 😄